service/ExchangeService.js

```diff
@@ -14,12 +14,15 @@ var fetchExchangeData = function(exchange, url, callback){
 			try {
 				callback(null, JSON.parse(body));
 			}catch (e) {
-				return callback('ERROR parsing response from ' + exchange + '. Details: ' + e + '\nResponse::' + body);
+				log.debug('ERROR parsing response from ' + exchange + '. Details: ' + e + '\nResponse::' + body);
+				return callback(null, null);
```

Comment:
We don't want to return an error message in the callback?

Reply:
It's easy way to keep our API alive all the time. Because some of API services instead of response with JSON responding with .html and .xml then it breaks whole our service.

Reply:
Are we expecting callback to return errors as JSON? I assume error messages should be handled as string regardless, so it shouldn't break the service, though the message might be ugly. If string data is breaking then shouldn't it be escaped or properly handled?

The reason is that `return callback(null, null);` gives no information as to whether the call failed or succeeded, so doesn't seem like a very good practice to set it up this way.